### DISCUSSION TITLE
Relationship with Linuxserver

### DISCUSSION THREAD

**Discussion by mirobartovic** [EmulatorJS/EmulatorJS#260] [General]
@ethanaobrien What's your relationship with Linuxserver people and their Emulatorjs docker image?
https://github.com/linuxserver/emulatorjs

**Comment by ethanaobrien** [comment]
Our projects are both independent, but he is where I got the inspiration for the beta cores and I have sent him small snippets of code for things like speeding up save states


See https://github.com/linuxserver/emulatorjs/issues/24
